azure-pipelines/build/templates/doclient-lite-docker-steps.yml

```diff
@@ -2,18 +2,32 @@
 # Consume this steps template in one or more jobs by passing in parameter values.
 
 parameters:
-- name: targetOsArch    # example: debian10-arm32
+- name: targetOS        # example: ubuntu1804
```

Comment:
>targetOS [](http://example.com/codeflow?start=8&length=8)

targetOS seems to be unused, no?

Reply:
In all the docker steps templates.

Reply:
Thanks good catch - after removing the install script steps, targetOS is now unused. Need to remove from here as well as the non-template yaml for docker as well.